import util.file_utils as file_utils

file_utils.s3_call_with_error_handling('http://localhost:8080/something', None)